<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preference</class>
 <widget class="QDialog" name="Preference">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>300</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Script Collection</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>29</y>
           <width>551</width>
           <height>52</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Path:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_ScriptPath">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_ScriptBrowser">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_valid">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>581</width>
          <height>81</height>
         </rect>
        </property>
        <property name="title">
         <string>Terminal</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>561</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0">
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_DefaultTerminalApp">
            <item>
             <property name="text">
              <string>Decided by App</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_AppBrowser">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Open As:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Interface</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Language:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_InterfaceLanguage">
           <item>
            <property name="text">
             <string>Decided by System</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Themes:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_InterfaceThemes">
           <item>
            <property name="text">
             <string>Decided by System</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Window:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_InterfaceWindow">
           <item>
            <property name="text">
             <string>Decided by System</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="optionBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
